app/assets/stylesheets/application.scss

```diff
@@ -13,5 +13,17 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
-
  */
+ $navbargreen: #1b2b0d;
+
+ body {
+   padding-top: 60px;
+ }
+
+ #body-home-page {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 1 spaces